content/en/guides/core/registry/configure_registry.md

```diff
@@ -39,89 +39,132 @@ A registry admin can remove individual users or entire teams from a registry. To
 Removing a user from a team also removes that user's access to the registry.
 {{% /alert %}}
 
-## Registry roles
+### Change the owner of a registry
 
-Each user in a registry has a *registry role*, which determines what they can do in that registry. 
+A Registry admin can designate any member as a registry's owner, including a **Restricted Viewer** or a **Viewer**. Registry ownership is primarily for accountability purposes and does not confer any additional permissions beyond those granted by the user's assigned role.
 
-W&B automatically assigns a default registry role to a user or team when they are added to a registry. 
+To change the owner:
+1. Navigate to the Registry at https://wandb.ai/registry/.
```

Comment:
Confirming specifying product URLs in the docs. (In the past you suggested I remove them).